server/models/Solution.js

```diff
@@ -0,0 +1,27 @@
+module.exports = (sequelize, DataTypes) => {
+    const Solution = sequelize.define('Solution', {
```

Comment:
Solution would have a description, title, and diagram image. We will probably also embed draw.io live diagrams, but dont worry about it for now.

Reply:
Would solution also have the number of upvotes it has?

Reply:
+1

Reply:
> Would solution also have the number of upvotes it has?

I wouldn't worry about that for now. We can add that in later. Upvotes are more for comments.